src/main.rs

```diff
@@ -31,29 +31,74 @@ fn try_reload_config(config: &mut Config, config_file: &String) {
     }
 }
 
+///// Start parsing client input
+
 fn _parse_client_cmd(raw: &String) -> Result<String> {
     if let Some(cmd) = raw.split_whitespace().next() {
+        Ok(cmd.to_string())
+    } else {
+        Err(Error::WrongClientInputFormat)
+	}
+}
+
+fn _parse_client_arg(raw: &String) -> Result<String> {
+    if let Some(cmd) = raw.split_whitespace().skip(1).next() {
         let index = cmd.find(":");
         if index.is_some() {
-            // TODO!          // let mut split_cmd =
-  
-            Ok(cmd.to_string())
+            let split_cmd = &cmd[0..index.unwrap()];
+            println!("{:?}", split_cmd.to_string());
+            Ok(split_cmd.to_string())
         } else {
+            println!("{:?}", cmd.to_string());
             Ok(cmd.to_string())
         }
     } else {
-        Err(Error::FieldCommandIsNotSet)
+        Err(Error::WrongClientInputFormat) // repondre au client
     }
 }
 
-fn _parse_client_arg(raw: &String) -> Result<String> {
+fn _parse_client_process(raw: &String, client_arg: &String) -> Result<Option<u32>> {
```

Comment:
Like above, underscore and function name (specify we are parsing the process number)